tpl/toolbox/report.tpl.php

```diff
@@ -64,6 +64,21 @@
 					<textarea id="litespeed-report" rows="20" cols="100" readonly><?php echo esc_textarea($report); ?></textarea>
 				</td>
 			</tr>
+			<tr>
+				<th>&nbsp;</th>
+				<td>
+					<?php
+					$this->build_checkbox(
+						'attach_php',
+						sprintf(
+							__('Attach PHP info to report. It will use data returned by function <a %s>phpinfo()</a>, sections <code>INFO_GENERAL, INFO_CONFIGURATION, INFO_MODULES</code>.', 'litespeed-cache'),
```

Comment:
Did you get Lisa proof reading? 
Besides, all the code/func related words shouldn't be in translation. Move the whole part into %s plz. If there are more than 1 placeholder, use %2$s to avoid disorder in diff langs.

Reply:
Changed